packages/mdc-top-app-bar/README.md

```diff
@@ -149,9 +159,15 @@ Class | Description
 --- | ---
 `mdc-top-app-bar` | Mandatory.
 `mdc-top-app-bar--fixed` | Class used to style the top app bar as a fixed top app bar.
+`mdc-top-app-bar--fixed-adjust` | Class used to style the content below the standard and fixed top app bar to prevent it from appearing below the top app bar.
```

Comment:
General wording suggestion: would "to prevent the top app bar from covering it" be clearer?

Right now it reads sort of like "it's the content below the bar, but we don't want it to appear below the bar...wait, what?"